src/pages/SplashScreen.js

```diff
@@ -15,7 +15,7 @@ export default function SplashScreen() {
   if (Config.get(ConfigKeys.state.setup_completed, false)) {
     console.log('Setup complete');
     setTimeout(() => {
-      Navigate('/manager');
+      Navigate('/home');
```

Comment:
We could change this string to use a value of `AllRoutes` which I recently exported from the [Routes.js](https://github.com/MSFS-Mega-Pack/MSFS2020-livery-manager/blob/217a26670bdb8a24e912458283cf48d4afd03f88/src/data/Routes.js#L7)

It'd prevent issues like this in the future.

Reply:
Changed 😄